cfspopcon/variables.yaml

```diff
@@ -1697,6 +1701,13 @@ SOC_LOC_ratio:
   set_by:
   - switch_to_linearised_ohmic_confinement_below_threshold
   used_by: []
+SOL_conduction_fraction:
+  default_units: dimensionless
+  description:
+  - Fraction of power carried by heat conduction in the scrape-off-layer.
+  set_by: []
+  used_by:
+  - calc_separatrix_electron_temp
```

Comment:
If it isn't `set_by` anything, where does `calc_separatrix_electron_temp` get this quantity from?

Reply:
Nevermind, I missed the fact that it's set to a default value 🙈